site/content/blog/2022-10-01-zap-and-hacktoberfest/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "ZAP and Hacktoberfest"
+summary: >
+    ZAP is partcipating in Hacktoberfest 2022.
+images:
+- https://www.zaproxy.org/site/blog/2022-10-01-zap-and-hacktoberfest/images/Hfest-Badge-2-Color-Void@2x.png
+type: post
+tags:
+- blog
+- hacktoberfest
+date: "2022-10-01"
+addSocialPreview: true
+authors:
+- arkaprabha
+---
+![Hacktoberfest X ZAP](images/zap_Hacktoberfest.png)
+
+We are happy to announce that ZAP is participating in [Hacktoberfest](https://hacktoberfest.com/) 2022.
+
+## Contributors
+This is what contributors need to know to participate and complete Hacktoberfest 2022:
+- Register anytime between September 26 and October 31.
+- Contributions/pull requests can be made to the following repositories under the [OWASP ZAP](https://github.com/zaproxy) organization:
+    - ZAP Core: [link](https://github.com/zaproxy/zaproxy)
+    - ZAP Extensions: [link](https://github.com/zaproxy/zap-extensions)
+    - ZAP HUD: [link](https://github.com/zaproxy/zap-hud)
+    - ZAP Core Help: [link](https://github.com/zaproxy/zap-core-help)
+    - ZAP Community Scripts: [link](https://github.com/zaproxy/community-scripts)
+    - ZAP Website: [link](https://github.com/zaproxy/zaproxy-website)
+    - ZAP API Docs: [link](https://github.com/zaproxy/zap-api-docs)
```

Comment:
What contributions are expected for these?

Reply:
Uhh, maybe grammatical corrections and detailed descriptions for certain areas?

Reply:
Any documentation issues or advanced examples :). Let's keep it open for contributors and see what they can add.. spam PRs would definitely be marked with a "spam" label.

Reply:
Most of the content is generated for other repos/sources.

Reply:
agreed

Reply:
I saw two enhancement issues. So thought of including the repo link.

Reply:
Which issues?

Reply:
[this](https://github.com/zaproxy/zap-api-docs/issues/88) and [this](https://github.com/zaproxy/zap-api-docs/issues/42).

Reply:
:)

Reply:
Neither of those is a good first issue, both issues need to be updated to be more clear what's expected to be done.

Reply:
Yes ... Hacktoberfest doesn't necessarily revolve around good first.. literally, any issues can be solved and any PRs which aren't spammy and accepted with the label "hacktoberfest-accepted" counts as a submission.

Reply:
It's about the cost benefit, leading contributors to issues that need more to time/effort from everyone is less useful than lead them to issues that are faster to understand and address.

Reply:
true that! I did mention that in the blog... "enhancement" issues will be challenging

Reply:
"yourself" is not "everyone" though.

Reply:
Now I don't how to process this :)

Reply:
What should I do then? drop the link? :)

Reply:
The link**s**, it's about both highlighted repos. I'd prefer that we focused the contributions to main repos.